### PR TITLE
Add developer documentation

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,9 @@ project based on this template:
 ```
 npx degit graph-paper-org/app-template graph-paper-app
 ```
+
+## Development
+
+Your new project will include [developer documentation](docs/development.md) but
+not this README. You may want to create your own README file which links to the
+developer documentation.
```
Comment (openjck) [inline README.md:12]: The developer documentation could live in a README, but this README file wouldn't be especially useful to teams that are *using* this template and degit doesn't provide a way to provide an alternative README file.

File: docs/development.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+to double-check that everything works after running `npm run format` just in
+case.
+
+## Deployment
+
+Run `npm install` to install all dependencies.
+
+Run `npm run build` to build this project for production deployment. After it's
+built, host the _public_ directory on a static server or use `npm start` to
+serve it with Node.
```
Comment (wlach) [inline docs/development.md:33]:
```suggestion
As for development, run `npm install` to install all dependencies. 

Then, run `npm run build` to build this project for production deployment. After it's built, host the _public_ directory on a static server or use `npm start` to serve it with Node.
```

File: docs/development.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+## Development
+
+Run `npm install` to install all dependencies.
+
+Run `npm run dev` to run this project in development mode. Run
+`npm run storybook` to run Storybook, which demonstrates how components can be
+used.
```
Comment (wlach) [inline docs/development.md:7]:
```suggestion
First, run `npm install` to install all dependencies.

Then, run `npm run dev` to run this project in development mode. Run
`npm run storybook` to run Storybook, which demonstrates how components can be
used.
```

File: docs/development.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+### Code quality
+
+To automatically benefit from the code quality tools that are included with this
+project, install editor plugins for [EditorConfig](https://editorconfig.org/),
```
Comment (wlach) [inline docs/development.md:12]:
```suggestion
project, use an editor (such as [Visual Studio Code](https://code.visualstudio.com/)) with plugins for [EditorConfig](https://editorconfig.org/),
```

**Comment by openjck** [comment]
Good suggestions. Thank you, Will.

**Pull request merged**